examples/FFPB_KPFM/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+echo "Download Hartree Potentials"
+wget --no-check-certificate "https://zenodo.org/records/10563098/files/KPFM_hartree.tar.gz"
+
+echo "Extract Hartree Potentials"
+tar xzvf KPFM_hartree.tar.gz
+
+echo "Calculate Electrostatic Forces and Polarizability in External Field"
+ppafm-generate-elff -i LOCPOT_V0.xsf -t dz2 --KPFM_sample LOCPOT_Vz.xsf --KPFM_tip fit --Vref +0.1 --Rcore -1.0 --z0 0.0
```

Comment:
would you please rather put `-t dz2` into _paramas.ini_  thank you!

Reply:
No, sorry. Because it doesn't work. But good that you asked for it and made me test it. I should make a new issue about that.

Reply:
Submitted as issue #256